resources/documentation/multiple_icebergs.rst

```diff
@@ -177,48 +225,64 @@ Create file /etc/bluebanquise/inventory/cluster/icebergs/iceberg2, with the foll
+Configure a new management, called mngt2, that will be in charge of iceberg2.
+According to icebergs definition, mngt2 must be connected to both ice1-1 network
+and ice2-1 network.
 
-For convenience, create a dedicated folder in cluster directory to store all nodes related to each iceberg, and move current nodes into iceberg1 directory.
+For convenience, we create a dedicated folder in cluster directory to store all
+nodes related to each iceberg. We also move all current nodes into iceberg1
+directory.
 
 .. code-block:: text
 
   mkdir -p /etc/bluebanquise/inventory/cluster/nodes/iceberg1/
-  mkdir /etc/bluebanquise/inventory/cluster/nodes/iceberg2/
+  mkdir -p /etc/bluebanquise/inventory/cluster/nodes/iceberg2/
   mv /etc/bluebanquise/inventory/cluster/*.yml /etc/bluebanquise/inventory/cluster/nodes/iceberg1/
 
-A warning may be displayed during playbook execution for now, because nodes/iceberg2/ is still empty.
+A warning may be displayed during playbook execution for now, because
+*nodes/iceberg2/* is still empty.
 
-Now create management2 file /etc/bluebanquise/inventory/cluster/nodes/iceberg2/management.yml:
+Now create mngt2 file dedicated file
+*inventory/cluster/nodes/iceberg2/management.yml* with the following content:
 
 .. code-block:: yaml
 
   mg_managements:
     children:
       equipment_typeM:
         hosts:
-          management2:
+          mngt2:
             bmc:
-              name: bmanagement2
+              name: bmngt2
               ip4: 10.10.100.2
               mac: 08:00:27:0d:41:97
               network: ice1-1
             network_interfaces:
-              enp0s3:
-                ip4: 10.10.0.2
-                mac: 08:00:27:de:41:21
-                network: ice1-1
-              enp0s8:
+              - interface: enp0s8
                 ip4: 10.12.0.1
                 mac: 08:00:27:de:42:23
                 network: ice2-1
+              - interface: enp0s3
+                ip4: 10.10.0.2
+                mac: 08:00:27:de:41:21
+                network: ice1-1
 
-This host is connected to both icebergs, and will be pushed from ice1-1 and act as a pusher (management) on ice2-1.
+This host is connected to both icebergs, and will be pushed from ice1-1 and act
+as a pusher (management) on ice2-1.
 
-And add management2 to iceberg2, by editing /etc/bluebanquise/inventory/cluster/icebergs/iceberg2 and adding management2 under [iceberg2]:
+.. warning::
+  Two important things, related to network.
+  First, BMC is connected to ice1-1, as mngt1 is in charge of deploying mngt2.
+  Secondly, here, network_interface on ice2-1 is put **FIRST** in the list. This
+  is key, as you need nodes to reach mngt2 to its main iceberg interface. Only
+  mngt1 should need access to ice1-1 interface of mngt2, and the *ssh_master*
+  role will ensure that Ansible from mngt1 use this one.
+
+And add mngt2 to iceberg2, by editing µinventory/cluster/icebergs/iceberg2µ and
```

Comment:
Remove the leading "And"? Add mngt2...
Typo around the path.

Reply:
good catch, corrected.